src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/PrisonConfigService.kt

```diff
@@ -115,4 +134,32 @@ class PrisonConfigService(
   private fun getExistingExcludeDates(prison: Prison): Set<LocalDate> {
     return prison.excludeDates.stream().map { it.excludeDate }.collect(Collectors.toSet())
   }
+
+  private fun validatePrisonDetails(
+    updatePolicyNoticeDaysMin: Int,
+    policyNoticeDaysMin: Int,
+    policyNoticeDaysMax: Int,
+    prisonCode: String,
+  ) {
+    if (updatePolicyNoticeDaysMin > policyNoticeDaysMin) {
```

Comment:
Need to write a test for these validations, also min value 0 might be illegal on normal min but not update

Reply:
added this validation now